docs/types.rst

```diff
@@ -752,11 +752,11 @@ Members
 
 **length**:
     Fixed-length arrays (storage or memory) have a read-only ``length`` member to access their number of elements.
-    Dynamic-sided arrays (only available for storage) have a read-write ``length`` member to resize the arary. Increasing the length adds unitialized elements to the array, this is O(1) complexity. Reducing the length performs ``delete()`` on each removed element and is O(n) complexity proportional to the number of elements being deleted. Please note that calling ``length--`` on an empty array will set the length of the array to 2^256-1 due to uint256 underflow wrapping.
+    Dynamically-sized arrays (only available for storage) have a read-write ``length`` member to resize the array. Increasing the length adds uninitialized elements to the array, this is O(1) complexity. Reducing the length performs ``delete`` on each removed element and is O(n) complexity where n is the number of elements being deleted. Please note that calling ``length--`` on an empty array will set the length of the array to 2^256-1 due to uint256 underflow wrapping.
```

Comment:
Is it "is" or "has" (complexity)?

Reply:
Fixed in 61286b341a63146b1dd943dae251c38da0280666